packages/@webex/webex-core/src/lib/webex-http-error.js

```diff
@@ -22,6 +22,15 @@ export default class WebexHttpError extends HttpError {
       value: res.options,
     });
 
+    Reflect.defineProperty(this, 'body', {
```

Comment:
what is this change made for?

Reply:
This change is added to get the error code if available from the failed API response so that we can log it for better troubleshooting.

Reply:
I'm kind of concerned about this change. Not sure why we have not needed this before. But I have not tested it and if it's needed, it does seem fine.